wondrous-bot-admin/src/components/CreateTemplate/index.tsx

```diff
@@ -184,8 +188,25 @@ const CreateTemplate = ({
 
   const hasReferralStep = steps?.some((step) => step.type === TYPES.REFERRAL);
 
+  const componentOptions = useMemo(() => {
+    let defaultOptions = [...COMPONENT_OPTIONS];
+    if (activeOrg?.id in CUSTOM_INTEGRATIONS) {
+      const customIntegrations = CUSTOM_INTEGRATIONS[activeOrg?.id];
+      customIntegrations?.integrations.forEach((integration) => {
+        defaultOptions.push({ ...integration, icon: activeOrg?.profilePicture, category: COMPONENT_CATEGORIES.CUSTOM });
+      });
+    }
+    return [...defaultOptions];
```

Comment:
I think there's no need for spread operator here since you already cloned the COMPONENT_OPTIONS